### DISCUSSION TITLE
Query labels by exact names

### DISCUSSION THREAD

**Discussion by ENikS** [community/community#24555] [Apps, API and Webhooks]
<p>I am trying to run simple query</p>
<pre><code>{
  repository(name: "graphiql", owner: "graphql") {
    labels(first: 10, query: "plugin proposal") {
      totalCount
      edges {
        node {
          name
          color
        }
      }
    }
  }
}
</code></pre>
<p>Theoretically it should return one node, but it returns all partial matches:</p>
<pre><code>{
  "data": {
    "repository": {
      "labels": {
        "totalCount": 3,
        "edges": [
          {
            "node": {
              "name": "plugin proposal",
              "color": "fbca04"
            }
          },
          {
            "node": {
              "name": "proposal",
              "color": "1db4e2"
            }
          },
          {
            "node": {
              "name": "potential plugin",
              "color": "efd1ac"
            }
          }
        ]
      }
    }
</code></pre>
<p>Could someone point me to where I could get more info on how the query string must be composed?</p>

**Comment by ENikS** [comment, marked as answer]
<p>After spending coulple of days looking for an answer I stumbled upon it by accident. Looking at the url of the search UI gave me an idea:<br>
<a href="https://github.com/graphql/graphiql/labels?q=%22plugin+proposal%22+" rel="nofollow noopener">https://github.com/graphql/graphiql/labels?q=%22plugin+proposal%22+</a></p>
<p>So the correct query would be like this:</p>
<pre><code>{
  repository(name: "graphiql", owner: "graphql") {
    labels(query: "\"plugin+proposal\"+", first: 10) {
      nodes {
        name
        id
      }
    }
  }
}
</code></pre>

**Comment by AndreaGriffiths11** [comment]
<p>Awesome <a class="mention" href="/eniks">@eniks</a>! Thank you so much for coming back to share what worked for you!</p>
